src/mixing_fab_tests.h

```diff
@@ -0,0 +1,44 @@
+#ifndef MIXINGFAB_TESTS_H_
+#define MIXINGFAB_TESTS_H_
+
+#include <gtest/gtest.h>
+
+#include "mixing_fab.h"
+
+#include "context.h"
+#include "facility_tests.h"
+#include "agent_tests.h"
+
+namespace cycamore {
+  // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  class MixingFabTest : public ::testing::Test {
```

Comment:
You can keep this fixture class if you want, but move it into the cc file.  The separation just makes it more work to navigate test failures and figure out what is going on (IMO - reasonable people could disagree).